### PR TITLE
build: #6 deploy to central requirements

### PR DESCRIPTION
**Pull request by rdjurin**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,20 +4,61 @@
...
-    <groupId>com.github.balconyseats</groupId>
-    <artifactId>vertx-application-support</artifactId>
+    <groupId>io.github.balcony-seats</groupId>
+    <artifactId>vertx-helper</artifactId>
     <version>0.9.0</version>
+    <packaging>jar</packaging>
+
+    <name>${project.groupId}:${project.artifactId}</name>
+    <description>Vertx helper library</description>
+    <url>https://github.com/balcony-seats/vertx-application-support</url>
```
Comment (darko-poljak) [inline pom.xml:14]: Project repository name should also be changed to artifact id value 'vertx-helper'.

**Review by darko-poljak — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (rdjurin) [reply pom.xml]: Renamed

File: pom.xml [hunk 2]
```diff
@@ -4,20 +4,61 @@
...
+            <name>Darko Poljak</name>
+            <email>darko.poljak@gmail.com</email>
+            <url>https://github.com/darko-poljak</url>
+        </developer>
+    </developers>
+
+    <scm>
+        <connection>scm:git:git://github.com/balcony-seats/vertx-application-support</connection>
+        <developerConnection>scm:git:https://github.com/balcony-seats/vertx-application-support.git</developerConnection>
+        <url>https://github.com/balcony-seats/vertx-application-support/tree/master</url>
```
Comment (darko-poljak) [inline pom.xml:39]: `vertx-application-support -> vertx-helper`

**Review by darko-poljak — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (rdjurin) [reply pom.xml]: Changed.

**Review by darko-poljak — APPROVED** [review]

**Pull request merged**